src/event_loop.rs

```diff
@@ -490,6 +491,16 @@ impl ActiveEventLoop {
             platform: self.p.owned_display_handle(),
         }
     }
+
+    /// Returns the current cursor position in screen coordinates.
```

Comment:
Referencing https://github.com/rust-windowing/winit/pull/2648#pullrequestreview-1939258628.

I think we _thought_ this was on `Window`, and hence expected things to be in window coordinates, just like the coordinates for `set_cursor_position` (such that `window.set_cursor_position(window.cursor_position()?)?` is (roughly) a no-op). But we decided against `Window::cursor_position` in https://github.com/rust-windowing/winit/pull/2648/files#r1090072945, this API is `ActiveEventLoop::cursor_position`.

In that case, I'd expect the coordinates to be the same as those from `Window::outer_position`, is that correct? If so, could you refer to that for discussion about the coordinate space, just like `Window::set_outer_position` does?

Reply:
(If you think that this interpretation is correct, I'll test the macOS implementation myself to ensure that it matches `Window::outer_position`).

Reply:
> In that case, I'd expect the coordinates to be the same as those from `Window::outer_position`, is that correct?

Yeah

> If so, could you refer to that for discussion about the coordinate space, just like `Window::set_outer_position` does?

like this https://github.com/amrbashir/winit/commit/1f9a3df3ff39eca403d0963dbee72267fdb29a2b ?

Reply:
Yup.

And I've tested it now, and pushed a fix to return positions with the correct scaling and such (it's still a little broken, but that's a more general issue, see https://github.com/rust-windowing/winit/issues/3615).